src/test/java/easytests/api/v1/controllers/SubjectsControllerTest.java

```diff
@@ -139,8 +141,38 @@ public void testListForbidden() throws Exception {
     /**
      * show(subjectId)
      */
-    /**
-     * delete(subjectId)
-     */
-    
+    @Test
+    public void testDeleteSuccess() throws Exception {
+        final SubjectModelInterface subjectModel = this.subjectsSupport.getModelFixtureMock(0);
+        when(this.subjectsService.find(any(), any())).thenReturn(subjectModel);
```

Comment:
also we need to be ensure that object with specified ID passed into delete method of service.